assets/app/view/game/company.rb

```diff
@@ -34,6 +36,11 @@ def select_company(event)
           return process_action(Engine::Action::Assign.new(entity, target: selected_company))
         end
 
+        if @game.round.actions_for(entity).include?('select_multiple_companies') && @step.respond_to?(:select_company)
```

Comment:
You don't need the `respond_to?` check here, any game that wants to have a `select_multiple_companies` action will have to implement this method.
```suggestion
        if @game.round.actions_for(entity).include?('select_multiple_companies')
```